File: CyberSecurity/Python/system_admin.py
from employee import Employee

class SystemAdmin(Employee):
    def __init__(self, name, e_id, department, admin_level="Junior"):
        super().__init__(name, department, e_id, "2022-01-01")
        self.admin_level = admin_level
        self.access_level = "Admin"

    def register_ip(self, ip):
        print(f"SystemAdmin {self.name} registered IP: {ip} with admin privileges")

    def monitor_employee(self, emp_name):
        print(f"SystemAdmin {self.name} monitoring employee: {emp_name}")

    def delete_system(self, system_name):
        if self.admin_level.lower() == "senior":
            print(f"SystemAdmin {self.name} deleted system: {system_name}")
        else:
            print(f"SystemAdmin {self.name} does not have permission to delete system: {system_name}")

File: CyberSecurity/Java/SystemAdmin.java
public class SystemAdmin extends Employee {
    private String adminLevel;

    public SystemAdmin() {
        super();
        this.adminLevel = "Junior";
        this.accessLevel = "Admin";
    }

    public SystemAdmin(String name) {
        super(name);
        this.adminLevel = "Senior";
        this.accessLevel = "Admin";
    }

    public SystemAdmin(String name, String id) {
        super(name, id, "IT");
        this.adminLevel = "Senior";
        this.accessLevel = "Admin";
    }

    public void getAdminInfo(){
        System.out.println("Admin Name: " + name + ", Admin Level: " + adminLevel + ", Access Level: " + accessLevel);
    }
    public void registerIP(String ip) {
        System.out.println("SystemAdmin " + name + " registered IP: " + ip + " with admin privileges");
    }

    public void monitorEmployee(String empName) {
        System.out.println("SystemAdmin " + name + " monitoring employee: " + empName);
    }

    public void deleteSystem(String systemName) {
        System.out.println("SystemAdmin " + name + " deleted system: " + systemName);
    }
}